### PR TITLE
RFC Process

### PR DESCRIPTION
**Pull request by TelegramSam**
Signed-off-by: Sam Curren <telegramsam@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+An accepted RFC is a standards-track document. It becomes an acknowledged
+standard when there is evidence that the community is deriving meaningful
+value from it. So:
+
+- Implement the ideas, and find out who else is implementing.
+- Socialize the ideas. Use them in other RFCs and documentation.
+- Update the agent test suite to reflect the ideas.
+
+When you believe an RFC is a _de facto_ standard, raise a PR that changes the status to ADOPTED. 
+the __adopted__ status. If the community is friendly to the idea, the doc
```
Comment (dhh1128) [inline README.md]: There's a little glitch here with an incomplete sentence. Just needs wordsmithing.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+#### License
+
+This repository is licensed under an [Apache 2 License](LICENSE).
+
+#### Contributions
+
+Unless you explicitly state otherwise, any contribution intentionally submitted
+for inclusion in the repo by you, as defined in the Apache-2.0 license, shall be
+dual licensed as above, without any additional terms or conditions.
```
Comment (troyronda) [inline README.md]: I see the Apache License above. What do you mean by dual licensed?

File: README.md (hunk 1 above)
Comment (TelegramSam) [reply README.md]: fixed. thanks

File: README.md (hunk 2 above)
Comment (TelegramSam) [reply README.md]: typo from source document. fixed. thanks!

**Pull request merged**